Server/Models/chat_message.js

```diff
@@ -29,10 +34,17 @@ ChatMessage.init({
     type: DataTypes.TEXT,
     allowNull: false,
   },
+  createdat: {
+    type: DataTypes.DATE,
+    defaultValue: DataTypes.NOW,
+    field: 'createdat',
+  },
```

Comment:
this field is not necessary because the ORM integrates that from itself, the ORM create and manages `createat`and `updatedat` by it self